README.md

```diff
@@ -181,7 +181,7 @@ make
 Then as `root` user or prefixed with `sudo`:
 
 ```console
-make PREFIX=/usr/local install
+make DESTDIR=/usr/local/bin install
```

Comment:
```suggestion
make PREFIX=/usr/local install
```

Reply:
I propose to revert this because IMHO because it's not really compliant with: https://www.gnu.org/prep/standards/html_node/DESTDIR.html